src/js/disability-benefits/526EZ/components/ReviewCardField.jsx

```diff
@@ -0,0 +1,189 @@
+      properties = Object.keys(schema.items.properties);
+    } else {
+      // Shouldn't get here, but in case we update the gate in the constructor later...
+      throw new Error(`Unknown schema type in ReviewCardField: ${schema.type}`);
+    }
+
+    const title = this.getTitle();
+
+    return (
+      <div className="review-card">
+        <div className="review-card--body input-section va-growable-background">
+          <h4>{title}</h4>
+          {properties.map(propName => {
+            return (
+              <div key={propName}>
+                <SchemaField
+                  name={propName}
+                  required={this.isRequired(propName)}
+                  schema={schema.properties[propName]}
+                  uiSchema={uiSchema[propName]}
+                  errorSchema={errorSchema[propName]}
+                  idSchema={idSchema[propName]}
+                  formData={formData[propName]}
+                  onChange={this.onPropertyChange(propName)}
+                  onBlur={onBlur}
+                  registry={registry}
+                  disabled={disabled}
+                  readonly={readonly}/>
+              </div>
+            );
+          })}
+          <button className="usa-button-primary update-button" onClick={this.update}>Update</button>
+        </div>
+      </div>
+    );
+  }
+
+
+  getTitle = () => {
+    const { uiSchema, formData } = this.props;
+    return typeof uiSchema['ui:title'] === 'function'
+      ? uiSchema['ui:title'](formData)
+      : uiSchema['ui:title'];
+  }
+
+
+  getReviewView = () => {
+    const ViewComponent = this.props.uiSchema['ui:options'].viewComponent;
+    const title = this.getTitle();
+
+    return (
+      <div className="review-card">
+        <div className="review-card--header">
+          <h4>{title}</h4>
+          <button className="usa-button-secondary" onClick={this.startEditing}>Edit</button>
+        </div>
+        <div className="review-card--body">
+          <ViewComponent formData={this.props.formData}/>
+        </div>
+      </div>
+    );
+  }
+
+
+  startEditing = () => {
+    this.setState({ editing: true });
+  }
+
+
+  isRequired = (name) => {
+    const { schema } = this.props;
+    const schemaRequired = Array.isArray(schema.required) &&
+      schema.required.indexOf(name) !== -1;
+
+    if (schemaRequired) {
+      return schemaRequired;
+    }
+
+    return false;
+  }
+
+
+  update = (event) => {
+    // Don't act like the continue button
+    if (event) {
+      // Apparently the unit tests don't send this event to the onClick handler
+      event.preventDefault();
+    }
+
+    if (!errorSchemaIsValid(this.props.errorSchema)) {
+      // Show validation errors
+      this.props.formContext.onError();
+    } else {
+      this.setState({ editing: false });
+    }
+  }
+
+
+  render() {
+    if (this.state.editing) {
+      return this.getEditView();
+    }
+
+    return this.getReviewView();
+  }
+}
+
```

Comment:
This sucker could probably use some `propTypes` given all the props used, if you're up for it 😬

Reply:
Eh...yeah...I started with them, but removed 'em because they were incomplete. I'll take a stab at putting all the props we're using in.

...I probably won't take a stab at outlining their structure. :grimacing: